docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -996,85 +1020,91 @@ More useful methods will be added to it over time.
+This property should be set to a value that is less than the ID token lifespan; the closer it is to this lifespan value, the more often the ID token is refreshed.
 
-You can further optimize this process by having a simple JavaScript function ping your Quarkus endpoint to periodically emulate the user activity, which minimizes the time frame during which the user might have to be re-authenticated.
+You can further optimize this process by having a simple JavaScript function ping your Quarkus endpoint periodically to emulate the user activity, which minimizes the time frame during which the user might have to be re-authenticated.
 
 [NOTE]
---
+====
 You cannot extend the user session indefinitely.
 The returning user with the expired ID token will have to re-authenticate at the OIDC provider endpoint once the refresh token has expired.
---
+====
 
 [[oauth2]]
 === Integration with GitHub and non-OIDC OAuth2 providers
 
-Some well known providers such as GitHub or LinkedIn are not OpenID Connect but OAuth2 providers which support the `authorization code flow`, for example, link:https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps[GitHub OAuth2] and link:https://docs.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow[LinkedIn OAuth2].
+Some well-known providers such as GitHub or LinkedIn are not OpenID Connect providers, but OAuth2 providers that support the `authorization code flow`.
+For example, link:https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps[GitHub OAuth2] and link:https://docs.microsoft.com/en-us/linkedin/shared/authentication/authorization-code-flow[LinkedIn OAuth2].
 Remember, OIDC is built on top of OAuth2.
 
-The main difference between OpenID Connect and OAuth2 providers is that OpenID Connect providers return an `ID Token` that represents a user authentication, in addition to the standard authorization code flow `access` and `refresh` tokens returned by `OAuth2` providers.
+The main difference between OIDC and OAuth2 providers is that OIDC providers return an `ID Token` that represents a user authentication, in addition to the standard authorization code flow `access` and `refresh` tokens returned by `OAuth2` providers.
 
-OAuth2 providers such as GitHub do not return `IdToken`, the fact of the user authentication is implicit and is indirectly represented by the `access` token which represents an authenticated user authorizing the current Quarkus `web-app` application to access some data on behalf of the authenticated user.
+OAuth2 providers such as GitHub do not return `IdToken`.
+The user authentication is implicit and is indirectly represented by the `access` token, which represents an authenticated user authorizing the current Quarkus `web-app` application to access some data on behalf of the authenticated user.
```

Comment:
It makes it somehow disconnected from the previous one, as it is not obvious it applies only to that one or if it is a more general statement. IMHO, having a comma is OK, as far as I recall the idea of a sentence per new line was about separate sentences only

Reply:
Hi @sberyozkin, You're right about having separate sentences as a new sentence per line . In this case, the original sentence is very long, which throws a warning in Vale. I take your point though about possible disconnect between the sentence. What about the following proposal instead, i.e. splitting the sentence before the "which..."
"OAuth2 providers such as GitHub do not return `IdToken`, and the user authentication is implicit and indirectly represented by the `access` token.
**The `access` token represents** an authenticated user authorizing the current Quarkus `web-app` application to access some data on behalf of the authenticated user."

Reply:
Done.